examples/misc_ubiquity_test.html

```diff
@@ -51,8 +51,8 @@
 
 				// QRCODE
 
-				var loader = new THREE.JSONLoader();
-				loader.load( 'models/json/QRCode.json', function ( geometry) {
+				var loader = new THREE.BufferGeometryLoader();
+				loader.load( 'models/json/QRCode_buffergeometry.json', function ( geometry ) {
 
 					mesh = new THREE.Mesh( geometry, new THREE.MeshLambertMaterial( { vertexColors: THREE.FaceColors } ) );
```

Comment:
`FaceColors` do not make sense for `BufferGeometry`. You will have to track each renderer that uses `Projector.js` to see what hacks are implemented in that case.

Reply:
Setting `VertexColors` does work now but the change feels like a hack. `Projector` and the respective renderers do not have a concept for supporting real vertex colors. Any chances to implement this different?